app/representers/api/auth/series_representer.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+
+class Api::Auth::SeriesRepresenter < Api::SeriesRepresenter
+
+  # point to authorized stories (including unpublished)
+  link :stories do
+    {
+      href: "#{api_authorization_series_stories_path(represented)}{?page,per,zoom,filters}",
+      templated: true,
+      count: represented.all_stories.count
+    } if represented.id
+  end
+  embed :all_stories, as: :stories, paged: true, item_class: Story,
+                      item_decorator: Api::Auth::StoryMinRepresenter,
+                      url: ->(_r) { api_authorization_series_stories_path(represented.parent) },
+                      zoom: false
+
+  # TODO: why is this necessary? maybe because :all_stories != :stories ???
+  embed :stories, zoom: false
```

Comment:
Yeah, this is definitely working around a bug.  (That all_stories `embed` doesn't respect its own `zoom: false`).  But if we're not into using the `?zoom` param anyways, I can probably just take out the embed.

Although I may need something like this line, to override the `embed` from the parent representer.